testsuite/performance/load-balancer/wildfly-modcluster/src/main/scripts/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM jboss/base-jdk:8
+
+ENV JBOSS_HOME /opt/jboss/wildfly
+WORKDIR $JBOSS_HOME
+
+ENV CONFIGURATION standalone.xml
+
+# Ensure signals are forwarded to the JVM process correctly for graceful shutdown
+ENV LAUNCH_JBOSS_IN_BACKGROUND 1
+
+USER root
+RUN yum -y install iproute
```

Comment:
Please reshuffle the ADD/USER etc. to combine individual RUN commands into a single one: https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#minimize-the-number-of-layers

Reply:
This is intentional so that the `yum` command (which downloads stuff) doesn't have to be run on every change to the KC server settings.